### DISCUSSION TITLE
How to set minimum number of output tokens ?

### DISCUSSION THREAD

**Discussion by thangld201** [vllm-project/vllm#2451] [Q&A]
This param is equivalent to `min_new_tokens` in huggingface `.generate()`. Is this already supported in vLLM ? @WoosukKwon I only see the `max_tokens` option in `SamplingParam` ..

**Comment by simon-mo** [comment]
You can checkout this PR here https://github.com/vllm-project/vllm/pull/1945 but it has been stalled. If you can help getting this into a good shape that would be even better!
